### PR TITLE
Inital Vitepress setup

### PR DESCRIPTION
**Pull request by modmuss50**
Barebones vitepress setup.

### CODE REVIEW

File: .vitepress/.gitignore
Comment (IMB11) [inline .vitepress/.gitignore:5]: You should gitignore .vitepress/cache and .vitepress/dist

**Review by IMB11 — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS
Comment (IMB11) [inline .github/CODEOWNERS:1]: Not sure if this is intended, github doesn't like it. "BROKEN CODEOWNERS FILE"

**Review by IMB11 — CHANGES_REQUESTED** [review]
Just need to:

- [x] ~~Add the custom theme hook (copy over `.vitepress/theme` from here: https://github.com/IMB11/fabric-wiki/tree/master/.vitepress/theme)~~
- [x] ~~Enable `cleanUrls: true` in the config.~~
- [x] ~~Add srcExclude license, contributing + readme from site (optional)?~~

```ts 
srcExclude: [
    "README.md",
    "LICENSE.md",
    "CONTRIBUTING.md"
  ],
  ```

File: .github/CODEOWNERS
Comment (modmuss50) [reply .github/CODEOWNERS:1]: This was a copy from yarn, it might be ok once merged?

Comment (IMB11) [reply .github/CODEOWNERS:1]: Probably, could just be because its from a fork :+1:

**Comment by IMB11** [comment]
I can setup versioning in a seperate pull request if needed.

**Comment by modmuss50** [comment]
> I can setup versioning in a seperate pull request if needed.

Yes please, want to keep the PRs manageable 👍

**Review by IMB11 — DISMISSED** [review]

**Review by IMB11 — APPROVED** [review]

**Pull request merged**
